e2e-tests/native/test/pageobjects/page.ts

```diff
@@ -6,6 +6,6 @@ export class Page {
   }
 
   public async get(): Promise<true | void> {
-    return $(`~${this.pageId}`).waitForExist()
+    return $(`~${this.pageId}`).waitForExist({ timeout: 10000 })
```

Comment:
I can also undo this, it was causing the errors in approximately less then 10%, so we can also undo this, but as the tests take very long I think waiting one time 5s more, but prevent a retry is worth it, but I am open for discussions on this.

Reply:
Sounds good to me :+1: